lib/activerecord-import/import.rb

```diff
@@ -529,6 +529,7 @@ def import_associations(models, options)
       # notes:
       #    does not handle associations that reference themselves
       #    should probably take a hash to associations to follow.
+      return if models.nil?
```

Comment:
Do you know of a situation when `models` will be `nil`? I tried to write a test for this locally, but I couldn't see to find the right invocation to make it so.